### DISCUSSION TITLE
Account suspended and support ignores me

### DISCUSSION THREAD

**Discussion by ghost** [community/community#130822] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Hi @dmilburn and @adnamalin,

My GitHub account: https://github.com/hv33y was suspended.

I don't know why as I didn't receive any mails or hints indicating why this happened.
I was taking to some one here in the community trying to solve a problem when suddenly it logged me out and when tried to sign in a message appear that account suspended for violating GitHub’s terms of service. Can some one please what is happening?
I am so sad and can't describe my sadness. I never violated TOS in my life and if that happened it is absolutely a mistake and not intentional. I’m happy to oblige with all of GitHub’s policies and Terms.

Thank you!

**Comment by JohnCherukuri** [comment]
even my account got suspended for no reason... this my Ticket ID: 2869314

**Reply by xglooom** [reply]
Me too  Ticket ID: 2869495
Completely unexpected, just created a release in the repository and got banned.

**Reply by rafaelcetina** [reply]
me too ticket: 2869501

Just created a release in a private repository

**Reply by john-scansan** [reply]
Just got access to my account back...after 45 mins of suspension. Really disappointed and worried..

**Reply by iglooom** [reply]
Yep, confirm, I got back my account too.

**Comment by ghost** [comment]
My Ticket: [2867879]. Been more than 2 days, still didn't got back access to my account.

**Reply by ghost** [reply]
@dmilburn and @adnamalin. Please help me recover my account.

**Reply by ghost** [reply]
@queenofcorgis

**Reply by queenofcorgis** [reply]
Hi there,

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our Code of Conduct for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

**Comment by queenofcorgis** [comment, marked as answer]
The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
